src/lib.rs

```diff
@@ -68,7 +68,7 @@ pub mod author {
         }
 
         /// Extract the author information from the given line.
-        pub fn parse(author: &str) -> Result<Author, AuthorParseError> {
+        pub fn new(author: &str) -> Result<Author, AuthorParseError> {
```

Comment:
I think `new` should generally not be able to fail (return a Result). And since this literally parses the input string the old name should stay.

Reply:
The Rust book renamed its parsing function from Chapter 12 to `new` when moving it to the implementation of the `Config` struct.  Hence, I considered this procedure good style since it was shown in official introduction.  I will revert it.  :wrench:

Reply:
Yeah, the difference there is, that they do not return a `Result`. Generally, if there is a single "construct" method for your struct and it returns the struct, it would be called `new`.

Reply:
:wrench: :heavy_check_mark: